test/helpers/testbot.js

```diff
@@ -25,10 +25,12 @@ module.exports = function createSbot(opts = {}) {
     .use(require('ssb-db2/core'))
     .use(require('ssb-classic'))
     .use(require('ssb-box2'))
+    //.use(require('ssb-db2/compat/db'))
```

Comment:
```suggestion
```

Preferably no commented code.

Reply:
@Powersource Feel free to press "commit suggestion" and "add suggestion to batch" for all these changes I'm recommending